python/ray/ml/train/integrations/huggingface/huggingface_trainer.py

```diff
@@ -37,7 +69,10 @@ class HuggingFaceTrainer(TorchTrainer):
     sampling, as the dataset will already be sharded.
 
     Hugging Face loggers will be automatically disabled, and the ``local_rank``
-    argument in ``TrainingArguments`` will be automatically set.
+    argument in ``TrainingArguments`` will be automatically set. Please note
+    that if you want to use CPU training, you will need to set the ``no_cuda``
+    argument in ``TrainingArguments`` manually - otherwise, an exception
+    may be thrown.
```

Comment:
Not without some really involved monkey patching

Reply:
It doesn't. In DDP mode, it will always try to use CUDA unless specified otherwise by the user. This was confirmed as intended behavior in https://github.com/huggingface/transformers/pull/16649

Reply:
Interesting...their description definitely misleading then.